### PR TITLE
feat(ENG_1093): remove grafana's pvc

### PR DESCRIPTION
**Pull request by MacLikorne**

### CODE REVIEW

File: test_utilities/src/aws.rs [hunk 1]
```diff
@@ -23,7 +23,7 @@ use crate::utilities::{build_platform_local_docker, FuncTestsSecrets};
 pub const AWS_REGION_FOR_S3: &str = "eu-west-3";
 pub const AWS_TEST_REGION: &str = "eu-west-3";
 pub const AWS_KUBERNETES_MAJOR_VERSION: u8 = 1;
-pub const AWS_KUBERNETES_MINOR_VERSION: u8 = 19;
+pub const AWS_KUBERNETES_MINOR_VERSION: u8 = 20;
```
Comment (benjaminch) [inline test_utilities/src/aws.rs:26]: maybe move this update into another PR?

File: test_utilities/src/scaleway.rs [hunk 2]
```diff
@@ -21,7 +21,7 @@ use tracing::error;
 
 pub const SCW_TEST_ZONE: ScwZone = ScwZone::Paris2;
 pub const SCW_KUBERNETES_MAJOR_VERSION: u8 = 1;
-pub const SCW_KUBERNETES_MINOR_VERSION: u8 = 19;
+pub const SCW_KUBERNETES_MINOR_VERSION: u8 = 20;
```
Comment (benjaminch) [inline test_utilities/src/scaleway.rs:24]: maybe move this update into another PR?

**Review by benjaminch — APPROVED** [review]

File: test_utilities/src/aws.rs (hunk 1 above)
Comment (MacLikorne) [reply test_utilities/src/aws.rs:26]: did this to avoid waiting for first pipeline due to version change + merge before doing this grafana change.

**Review by benjaminch — APPROVED** [review]

**Review by benjaminch — APPROVED** [review]

**Review by deimosfr — APPROVED** [review]

**Review by deimosfr — APPROVED** [review]
